include/crm/cluster.h

```diff
@@ -8,20 +8,16 @@
  */
 
 #ifndef PCMK__CRM_CLUSTER__H
-#  define PCMK__CRM_CLUSTER__H
+#define PCMK__CRM_CLUSTER__H
 
-#  include <stdint.h>           // uint32_t, uint64_t
-#  include <glib.h>             // gboolean, GHashTable
-#  include <libxml/tree.h>      // xmlNode
-#  include <crm/common/xml.h>
-#  include <crm/common/util.h>
+#include <glib.h>               // gpointer
```

Comment:
try compiling sbd with this to make sure it wasn't depending on any transitive includes